### PR TITLE
Add eclipse project files #43

### LINKED ISSUE #43: Add eclipse project files
as per title

### PR DESCRIPTION
**Pull request by m133225**
Fixes #43

### CODE REVIEW

**Comment by m133225** [comment]
@damithc Added a eclipse plugin for gradle and attempted to set it the project on eclipse. Do verify if they are working.

File: addressbook-level4.userlibraries [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <archive path="/Users/m133225/.gradle/caches/modules-2/files-2.1/org.testfx/openjfx-monocle/1.8.0_20/9859038affe2fd08c40c43ca19790687a5d56107/openjfx-monocle-1.8.0_20.jar" />
+  </library>
+  <library name="Gradle: org.testfx:testfx-core:4.0.4-alpha">
+    <archive path="/Users/m133225/.gradle/caches/modules-2/files-2.1/org.testfx/testfx-core/4.0.4-alpha/298beafb314304fd3243eb0bd9ac0450cb2e1941/testfx-core-4.0.4-alpha.jar" />
+  </library>
+  <library name="Gradle: org.testfx:testfx-junit:4.0.4-alpha">
+    <archive path="/Users/m133225/.gradle/caches/modules-2/files-2.1/org.testfx/testfx-junit/4.0.4-alpha/6bcea501014538d3d6c375dc5468915f8771d469/testfx-junit-4.0.4-alpha.jar" />
+  </library>
+  <library name="Gradle: org.testfx:testfx-legacy:4.0.4-alpha">
+    <archive path="/Users/m133225/.gradle/caches/modules-2/files-2.1/org.testfx/testfx-legacy/4.0.4-alpha/8cda3025db09491282aedec6427dc71c4b445cbf/testfx-legacy-4.0.4-alpha.jar" />
```
Comment (damithc) [inline addressbook-level4.userlibraries]: This has your username. Will not work for others?

**Comment by m133225** [comment]
Removed them, probably some incorrect files generated by IDEA's export functionality, does not seem to affect anything.

**Comment by m133225** [comment]
Tried clearing gradle cache before re-importing the project. Seems to download the dependencies upon import. A simple `Right-click` > `Gradle`> `Refresh` should also recheck the dependencies. (Can't verify the exact steps at the moment since it is taking rather long to re-download all gradle AND project dependencies.)

File: src/test/data/ConfigUtilTest/SomeFile.json [hunk 2]
```diff
@@ -0,0 +1 @@
+null
```
Comment (damithc) [inline src/test/data/ConfigUtilTest/SomeFile.json]: Are these two files needed?

**Comment by damithc** [comment]
Worked for me. Can go ahead and merge.

**Comment by damithc** [comment]
I'm getting a few random test failures though. Seem to pass upon rerunning. 
guitests.AddCommandTest
guitests.SelectCommandTest

**Comment by m133225** [comment]
Yes I think that happened when I first ran them too yesterday. Haven't looked through the test codes for that. Should I revert that commit?

**Comment by damithc** [comment]
Didn't happen with IDEA though. So may be Eclipse hogs more resources which slows down the GuiTests?

**Comment by damithc** [comment]
I've added some delays which seems to solve the problem, at least on my machine. Let's merge this so that I can get tutors to test it.

**Pull request merged**